README.md

```diff
@@ -317,24 +317,27 @@ class App extends React.Component {
 
 ```sh
 # -- build the package --
-npm i
-npm watch # or build it once with npm run build
+yarn i
+yarn build
 ```
 
 ```sh
 # -- test the package --
```

Comment:
If I understood the setup correctly, I'd make it more explicitly, that there is a example setup and that's how you test it:

```suggestion
# -- test the package in a dedicated example setup --
```